scripts/cluster

```diff
@@ -14,9 +14,8 @@ function initIbftConsensus() {
 
 function initPolybftConsensus() {
     echo "Running with polybft consensus"
-    genesis_params="--consensus polybft --bridge-json-rpc http://127.0.0.1:8545"
+    genesis_params="--consensus polybft"
```

Comment:
Don't we need the bridge RPC address?

Reply:
Nope, we don't need it eventually. :slightly_smiling_face:

The mindset was like this:
let's configure everything related to the rootchain in `rootchain deploy` command. So we are populating JSON RPC endpoint address in that particular command, as part of `RootchainConfig` (https://github.com/0xPolygon/polygon-edge/pull/1389/files#diff-db0b676aabd14c80bf4a9bfa6c9353fc37b51d3a75763cf23ebb59ea8ddf462aR301-R303), which is then converted into `BridgeConfig` (https://github.com/0xPolygon/polygon-edge/pull/1389/files#diff-db0b676aabd14c80bf4a9bfa6c9353fc37b51d3a75763cf23ebb59ea8ddf462aR207) and written into the genesis file (https://github.com/0xPolygon/polygon-edge/pull/1389/files#diff-db0b676aabd14c80bf4a9bfa6c9353fc37b51d3a75763cf23ebb59ea8ddf462aR223).